### ISSUE TITLE
left-click on touchpad works strange

### ISSUE THREAD

**Issue by sojus-31** [ghostbsd/issues#12]
### Version

23.07.29

### Explain the issue and what happens

i use a lenovo thinkpad t470p.
minimize a window or close a window works always with the touchpad-buttons.
maximize a window from the taskbar 95% dont work. during the push of the button the window is be maximized (for the time you hold the pressure on the button) and then it is going minimize again. the only way to maximize this window is using the right button and use "restore".

### Explain how to reproduce the bug?

the bug happens all the time.

### What is the Expected behavior (if applicable)

left and right touchpad-buttons works like a left and right mouse-buttons .)

**Comment by ericbsd** [comment]
That seems odd. I am not sure why but I can see if there is a similar issue reported to FreeBSD.

**Comment by sojus-31** [comment]
Maybe you dont found anything at FreeBSD. This issue only exists on GhostBSD. I start with GhostBSD+XFCE and found the issue (and some other). Then i installed FreeBSD+XFCE and dont got the issue. But i like a userfriendly system with softwarecenter and updatecenter. So i switched back and tried GhostBSD+Mate and run into the same issue again. :)

**Comment by ericbsd** [comment]
It might me something that is installed on GhostBSD that is not on FreeBSD.

I will make a review of what is installed and see what could cause that issue.

Oh do you have touch screen on that laptop?

**Comment by sojus-31** [comment]
no, it is only touchpad with extra mouse-buttons. the "double-click" at the touchpad for left mouse works fine. but i like to use the buttons. here you can found some laptop-pics: https://dazit.de/gebraucht-refurbished/notebooks/11864/notebook-lenovo-thinkpad-t470p-core-i7-7820hq-2-9ghz-refurb.a

**Comment by ericbsd** [comment]
On vanila FreeBSD did you setup `hw.psm.synaptics_support="1"` in /boot/loader.conf?

**Comment by ericbsd** [comment]
After looking deeper maybe `xf86-input-evdev` is to blame. If you remove it, it might solve your issue.

**Comment by sojus-31** [comment]
remove the lic-folder dont solve the issue. on vanilla - i only had trouble with nvidia at xorg and it was needed to install the extra driver. i dont touch any other config.

**Comment by ericbsd** [comment]
Did you try to remove `xf86-input-evdev` on GhostBSD?

**Comment by sojus-31** [comment]
like i wrote - remove the lic-folder dont solve the issue. i only found something like: /usr/local/share/licenses/xf86-input-evdev* and deleted it. that's why i called it the lic-folder.

**Comment by LainLayer** [comment]
I am having a similar issue with my touchpad buttons. Checking what is happening with `xev` it seems like in very quick succession, after mouse 1 is pressed, mouse 4 is then pressed. For mouse 3, it is followed by mouse 5, and for mouse 2, it is followed by mouse 6. Clicking the touchpad itself only produces a mouse 1 click. If i enable clicking with the touchpad by tapping, it also only produces mouse button 1 clicks.

I have tried deleting `xf86-input-evdev` and it did not help.

Computer: Thinkpad x270
Desktop Environment: MATE
